Gifski/EditVideoViewController.swift

```diff
@@ -299,6 +299,19 @@ final class EditVideoViewController: NSViewController {
 		}
 	}
 
+	private func showConversionCompletedAnimationWarningIfNeeded() {
+		// TODO: This function can be removed once NSImageView respects the looping counter in the GIF header.
```

Comment:
Can you mirror the FB you submitted to Apple over at https://github.com/feedback-assistant/reports (follow the template) and then link to it here?

Reply:
Done and thank you! I didn't know this resource existed.

Reply:
This comment is now outdated. The code should not be removed when it's fixed as it also applies to Quick Look. Rather the text should be modified to only mention Quick Look when it's fixed.